### PR TITLE
Create mk.conf.2016Q4

### PR DESCRIPTION
**Pull request by caius**

### CODE REVIEW

File: mk.conf.2016Q4 [hunk 1]
```diff
@@ -0,0 +1,22 @@
+.ifdef BSD_PKG_MK       # begin pkgsrc settings
+
+####################################
+# FreeAgent Specific Build Options #
+####################################
+
+JPEG_DEFAULT=libjpeg-turbo
```
Comment (stevenwilliamson) [inline mk.conf.2016Q4]: what this here for ?

Comment (caius) [inline mk.conf.2016Q4]: I copied it from the 2015Q4 one 😁

Comment (caius) [inline mk.conf.2016Q4]: https://github.com/fac/pkgsrc-fa/commit/86765acee28cf9a939fd3759a42241f8355557bb

It's the default according to joyent?

Comment (stevenwilliamson) [inline mk.conf.2016Q4]: ah not needed if it's the default then, i was more wondering why we were changing from the default

**Pull request merged**